README.md

```diff
@@ -0,0 +1,345 @@
+
+Eclipse Codewind provides the ability to create projects based on a variety of different template types.  These include IBM Cloud starters,Odo, and Appsody templates.  When used with Kabanero, you will only use Appsody templates for the Collections your company has enabled.  Today, there are collections for: Eclipse MicroProfile/Java EE, Springboot, Node.js, Node.js with Express, Node.js with Loopback.
+
+This guide describes developing a simple microservice using two different IDEs. Select the guide section based on your IDE of choice: 
+
+* [Developing with VS Code](#developing-with-vs-code)
+* [Developing with Eclipse](#developing-with-eclipse)
+
+## Developing with VS Code
+
+If you use Visual Studio Code (VS Code), you can use Codewind for VS Code to develop and debug your containerized projects from within VS Code using the workflow you already use today.
+
+### Prerequisite 
+
+Before you can develop a microservice with VS Code, you need to:
+
+* [Install Docker](https://docs.docker.com/install/) 
+    * **Note:** Make sure to install or upgrade to minimum Docker version 19.03. 
+* [Install VS Code](https://code.visualstudio.com/download)
+ 
+### Installing Codewind for VS Code
+
+The Codewind installation includes two parts:
+
+1. The VS Code extension installs when you install Codewind from the [VS Code Marketplace](https://marketplace.visualstudio.com/items?itemName=IBM.codewind) and click `Install`. 
+    * Or, go to `View->Extensions`, search for Codewind, and click `Install`. 
+2. The Codewind backend containers install after you click `Install` when you are prompted. After you click `Install`, the necessary images are downloaded from the internet. The download is approximately 1 GB.
+    * **Optional:** If you don’t click `Install` when the notification window first appears, you can access the notification again. Go to `View->Explorer`. Then click `Codewind` and hover the cursor over `Codewind` where there is a switch to turn Codewind on or off. Click the switch so that it is `On`. The notification window is displayed. 
+
+The following images are pulled. These images together form the Codewind backend:
+
+1. `eclipse/codewind-performance-amd64`
+2. `eclipse/codewind-pfe-amd64`
+
+### Configuring Codewind with application stack hubs
+
+Configure Codewind to use application stack hubs so you can focus exclusively on your code:
+
+1. Under the Explorer pane, select `Codewind`. 
+2. Expand `Codewind` by clicking the drop-down arrow.
+3. Right-click `Projects (Local)`.
+4. Select `Manage Template Sources`. 
+5. Enable application stack hubs. 
+
+You now have configured Codewind to application stack hubs and can 
+proceed to develop your microservice within Codewind.
+
+### Creating an Appsody project
+
+Appsody helps you develop containerized applications and removes the burden of managing the full software development stack. If you want more context about Appsody, visit the [Appsody welcome page](https://appsody.dev/docs). 
```

Comment:
... applications leveraging containers managed by the organization and curated for usage throughout the application lifecycle.  <- trying to restate the value of pointing out a set of curated containers that the developers just use.  I'm not excited about the exact words I chose; but I wanted to focus on the value of Kabanero rather than the "burden".